pkg/api/schemav2/schemav2.proto

```diff
@@ -149,6 +170,26 @@ service ImmuServiceV2 {
       tags: "documents";
     };
   } 
+ 
+  rpc DocumentAudit(DocumentAuditRequest) returns (DocumentAuditResponse) {
+    option (google.api.http) = {
+      post: "/documents/audit"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "documents";
+    };
+  } 
+ 
+  rpc DocumentProof(DocumentProofRequest) returns (schema.VerifiableTx) {
```

Comment:
return type should be DocumentProofResponse

Reply:
https://github.com/codenotary/immudb/pull/1546